src/test/java/com/spotify/folsom/KetamaIntegrationTest.java

```diff
@@ -70,6 +72,22 @@ public static void tearDownClass() throws Exception {
     binaryServers.stop();
   }
 
+  public static void allClientsConnected(final MemcacheClient<?> client)
+      throws Exception{
+    final CountDownLatch done = new CountDownLatch(1);
+
+    client.registerForConnectionChanges(new ConnectionChangeListener() {
```

Comment:
maybe this should use a future similar to ConnectFuture instead of a countdownlatch?